lib/mud_util.py

```diff
@@ -1,34 +1,46 @@
+from typing import List
+
+from game_data import players, mud, game, rooms
+from lib.models.player import Player
 
 # TODO: Tests
 
-def greet_players(player_list, mud):
-    for event in mud.get_new_players():
-        connected_player = event.client
+def handle_new_connections():
```

Comment:
These two functions, `handle_new_connections` and `handle_disconnects` are good, but I think they're name wrong, in a sense, because the conflate the concern of "connection", which is a server concern with game logic. Something like, `handle_player_join` or something makes it feel more natural